pkg/acs/calacs/Updates

```diff
@@ -49,10 +49,10 @@ Update for version 8.2.0 - 12-Aug-13 (PLL)
     acscte/getctesw.c
     calacs/acsinit.c
     calacs/calacs.c
-    calacs/calacs.h
+    calacs/acsinfo.h
```

Comment:
Good spot. I wasn't, at least not intentionally, at one point I attempted to disambiguate the dangerous case of having two ACSInfo structs. I gave up at some point and this must be the remainder of that alteration that I hadn't noticed was still present. Thanks.